src/plugins/dashboard/public/application/listing/dashboard_listing.test.tsx

```diff
@@ -166,7 +172,8 @@ describe('after fetch', () => {
 
   test('renders warning when listingLimit is exceeded', async () => {
     const services = makeDefaultServices();
-    services.savedObjects.settings.getListingLimit = () => 1;
+    pluginServices.getServices().dashboardCapabilities.showWriteControls = true;
+    pluginServices.getServices().settings.uiSettings.get = jest.fn().mockReturnValueOnce(1);
```

Comment:
This line caused the related snapshot to change, hence why I had to update the snapshots for this test to pass - however, this test doesn't even **check** for an error message as the description suggests, so not sure if it's even worth keeping this test around? The error message is handled in the `TableListView` component, which isn't even rendered out in the snapshot 🤷